src/test/regress/sql/create_drop_database_propagation.sql

```diff
@@ -541,6 +536,14 @@ REVOKE CONNECT ON DATABASE test_db FROM propagated_role;
 DROP DATABASE test_db;
 DROP ROLE propagated_role, non_propagated_role;
 
+--test that create database with oid option failed when citus.enable_create_database_propagation is on
```

Comment:
Added this test to fix code coverage in citus_deparseutils

Reply:
Actually this won't fix the coverage because of `EnsureSupportedCreateDatabaseCommand`
2 options here: ignore Codecov warnings, or remove your oid additions from this PR. What do you think? @gurkanindibay @onurctirtir